include/tester/core/Application.hpp

```diff
@@ -59,16 +59,17 @@ class Application
 
     bsoncxx::document::value getHistoryHeaders()
     {
-        return(_db -> getAllHeaders("Test-Logs"));
+        return _database->getAllHeaders(std::move(COLLECTION_NAME));
     }
 
-    bsoncxx::document::value getHistoryLog(const std::string &doc_id)
+    bsoncxx::document::value getHistoryLog(const std::string &docId)
     {
-        auto result = _db -> getLog("Test-Logs", doc_id);
+        auto result = _database->getLog(std::move(COLLECTION_NAME), std::move(docId));
         return result.value();
     }
 
-    std::shared_ptr<Database> _db;
-    std::shared_ptr<SoftphoneManager>_sm;
-    std::shared_ptr<VTCPServer> _vs;
+    std::shared_ptr<Database> _database;
+    std::shared_ptr<SoftphoneManager>_manager;
+    std::shared_ptr<VTCPServer> _server;
+    static constexpr auto COLLECTION_NAME = "Test-Logs";
```

Comment:
I believe that this should go in the `Database.hpp` file instead of being in the `Application.hpp` file. It is irrelevant here.

Reply:
done